File: brn.erl
#!/usr/bin/escript
-export([main/1]).

main(_) ->
    {_, Bd} = file:get_cwd(),
    {_, Dirs} = file:list_dir(Bd),
    lists:foreach(fun(Dir) ->
                          {_, Files} = file:list_dir(Dir),
                          lists:foreach(fun(File) ->
                                                Fop = io_lib:format("~s/~s/~s", [Bd, Dir, File]),
                                                Fnp = io_lib:format("~s/~s - ~s", [Bd, Dir, File]),
                                                file:rename(Fop, Fnp)
                                        end,Files)
                  end,Dirs).

File: brn.exs
#!/usr/bin/elixir
bd = System.cwd()
{_, dirs} = File.ls(bd)
for dir <- dirs do
    {_, files} = File.ls(dir)
    for file <- files do
      fop = "#{bd}/#{dir}/#{file}"
      fnp = "#{bd}/#{dir} - #{file}"
      File.rename(fop, fnp)
    end
end
